lib/node_runner.go

```diff
@@ -63,9 +65,17 @@ func NewNodeRunner(
 	return nr
 }
 
+type HttpRe struct{}
+
+func (r HttpRe) ResponseMessage(w http.ResponseWriter, o string) {
+	fmt.Fprintf(w, string(o))
+}
+
+func (r HttpRe) ReceiveMessage(*sebaknetwork.HTTP2Network, sebaknetwork.Message) {}
```

Comment:
IMHO,The func `ReceiveMessage`is no implementation such as ` t.ReceiveChannel() <- Message`

Reply:
You are the one who pass the test :)